piker/service/_ahab.py

```diff
@@ -322,37 +354,94 @@ async def open_ahabd(
         ) = ep_func(client)
         cntr = Container(dcntr)
 
-        with trio.move_on_after(start_timeout):
-            found = await cntr.process_logs_until(start_lambda)
+        conf: ChainMap[str, Any] = ChainMap(
```

Comment:
Just curious, why use a ChainMap here?

Reply:
Because it's easier then writing a lot of `dict.get(blah, default)` stuff and makes everything extensible for if we ever need to offer overrides of container configs for specific use cases..